### PR TITLE
Shaka and Video.js samples, documentation and test result

### PR DESCRIPTION
**Pull request by ecascardo**
- Common Baseline Setup

- Testing, Samples and Documentation for Video.js

- Testing, Samples and Documentation for Shaka

### CODE REVIEW

File: docs/shaka/results/issues.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+## Issue #1
+
+HLS wiht Widevine is not suported. It may be thatAzure  is not providing the EXT-X-Key in the HLS manifest.
```
Comment (johndeu) [inline docs/shaka/results/issues.md]: Typos "with"

Comment (johndeu) [reply docs/shaka/results/issues.md]: Typso "that Azure"

Comment (johndeu) [reply docs/shaka/results/issues.md]: Maybe state this as Azure Media Services does not yet support EXT-X-KEY in the manifest for Widevine with HLS. This features is currently on the backlog

File: docs/shaka/results/issues.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+**In conclusion, Shaka player is not compatible with the AES-128 content generated in Azure media services.**
+
+A link to a github issue to follow the status of this feature in Shaka: 
+
+https://github.com/google/shaka-player/issues/850
+
+------
+
+## Issue #4 DAHS CMAF on iOS
```
Comment (johndeu) [inline docs/shaka/results/issues.md]: typos DASH

File: docs/shaka/results/issues.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+On iOS 12+ through Apple's native HLS player. Shaka provides the same top-level API, but Shaka just set the video's src element to the manifest/media. So Shaka is dependent on the browser supporting the manifests. So DASH is not souported11
```
Comment (johndeu) [inline docs/shaka/results/issues.md]: typo - "Shaka just sets"

Comment (johndeu) [reply docs/shaka/results/issues.md]: typo - "souported11"

File: docs/shaka/results/issues.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+## Issue #6
+
+Shaka doesn't support PlayReady in HLS. We found that Shaka are working on it. The issues is reported in: https://github.com/google/shaka-player/issues/1145 
+
+------
+
+## Issue #7 Edge HLS LowLatency Clear
+
+We did not find an issues on Video.js GitHub.
```
Comment (johndeu) [inline docs/shaka/results/issues.md]: Typeo - "any issues"

File: docs/shaka/results/issues.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+------
+
+## Issue #8 Mac with Low Latency
+
+We did not find an issues on Video.js GitHub.
+
+------
+
```
Comment (johndeu) [inline docs/shaka/results/issues.md]: "any issues"

File: docs/shaka/results/issues.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+## Issue #11 DASH in Android with Firefox 
+
+We did not find an issues on Video.js GitHub.
+
+------
+
```
Comment (johndeu) [inline docs/shaka/results/issues.md:45]: "any issues"

**Review by johndeu — APPROVED** [review]
Some minor typos noted, but approving.

**Comment by johndeu** [comment]
Should I do a Squash and Merge or a Rebase and Merge? @ecascardo-sw ?

I noted a few typos, can you address those throughout the docs and ping me.

**Comment by ecascardo** [comment]
@johndeu Thanks for the feedback. We'll apply all the changes (along some other improvements) and let you know once it'is ready. Regarding the merge approach, both should work, "Rebase and Merge" might be probably better since we would preserve the commit history.

**Comment by johndeu** [comment]
Thanks @ecascardo-sw  - Should I leave this PR open for your edits then? Just ping me.

**Comment by ecascardo** [comment]
@johndeu  We are closing up some changes that will include your feedback between other improvements. This will be completed today and I'll ping you once the pull request is updated. Thanks.

**Comment by ecascardo** [comment]
Hi @johndeu the pull request is ready to be reviewed and merged.

**Pull request merged**